Can't resolve 'react/jsx-runtime' in storybook-xstate-addon dist

On attempting to run storybook I am presented with the following error after installing this addon for the first time: 

```
ERROR in ./node_modules/storybook-xstate-addon/dist/preset/manager.js
[storybook] Module not found: Error: Can't resolve 'react/jsx-runtime' in '\node_modules\storybook-xstate-addon\dist\preset'
```

I am using react and react-dom version 16.2.0 and @storybook/react 6.2.9

Apologies if this an issue with my config and not the addon itself but I thought it was worth raising as this addon looks amazing and I really want to be able to use it

--- a/README.md
+++ b/README.md
@@ -2,6 +2,12 @@
 
 ![](./preview.png)
 
+### Installation
+
+`npm install -D @storybook/addons storybook-xstate-addon @xstate/inspect @xstate/react`
+
+### Usage
+
 To use this in storybook simply add `addons: ["storybook-xstate-addon/preset"]` to your storybook config.
 
 If you want to enable the inspector in all stories, set the following in your `/.storybook/preview.js` file. With this setting on you can disable the inspector in certain stories

--- a/package.json
+++ b/package.json
@@ -44,13 +44,16 @@
     "@storybook/react": "^6.3.1",
     "@types/react-dom": "17.0.8",
     "@xstate/inspect": "^0.4.1",
+    "@xstate/react": "^1.3.2",
     "auto": "^10.29.3",
     "babel-loader": "^8.2.2",
     "concurrently": "^6.2.0",
     "prettier": "^2.3.2",
     "rimraf": "^3.0.2",
     "typescript": "^4.3.4",
-    "xstate": "^4.20.1"
+    "xstate": "^4.20.1",
+    "react": "^17.0.0",
+    "react-dom": "^17.0.0"
   },
   "peerDependencies": {
     "@storybook/addons": "^6.1.14",
@@ -59,10 +62,8 @@
     "@storybook/core-events": "^6.1.14",
     "@storybook/theming": "^6.1.14",
     "@xstate/inspect": "^0.4",
-    "xstate": "^4.18.0"
-  },
-  "dependencies": {
     "@xstate/react": "^1.3.2",
+    "xstate": "^4.18.0",
     "react": "^17.0.0",
     "react-dom": "^17.0.0"
   },

